salt/modules/cmdmod.py

```diff
@@ -215,6 +216,31 @@ def _gather_pillar(pillarenv, pillar_override):
     return ret
 
 
+def _check_avail(cmd):
+    '''
+    Check to see if the given command can be run
+    '''
+    bret = True
+    wret = False
+    if __salt__['config.get']('cmd_blacklist_glob', []):
```

Comment:
True, I will fix it. That built in "always use the same type" habit strikes again :)